### PR TITLE
PLAT-143785: Added screenshot tests for SwitchItem

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Added extra screenshot tests for SwitchItem

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
PLAT-143785

### Comments

Enact-DCO-1.0-Signed-off-by: Daniel Stoian daniel.stoian@lgepartner.com

### CODE REVIEW

File: tests/screenshot/apps/components/SwitchItem.js [hunk 1]
```diff
@@ -5,18 +5,34 @@ import {withConfig} from './utils';
...
 	<SwitchItem />,
+	<SwitchItem>Hello SwitchItem</SwitchItem>,
+	<SwitchItem selected>Hello SwitchItem</SwitchItem>,
+	<SwitchItem disabled>Hello SwitchItem</SwitchItem>,
+	<SwitchItem disabled selected>Hello SwitchItem</SwitchItem>,
+	<SwitchItem inline>Hello SwitchItem</SwitchItem>,
+	<SwitchItem inline selected>Hello SwitchItem</SwitchItem>,
 
 	// Icon slotAfter
 	<SwitchItem>SwitchItem<Icon slot="slotAfter">home</Icon></SwitchItem>,
```
Comment (stanca-pop-lgp) [inline tests/screenshot/apps/components/SwitchItem.js:17]: Maybe you can also add a test with slotBefore and one with label.

Comment (daniel-stoian-lgp) [reply tests/screenshot/apps/components/SwitchItem.js:17]: switchItem does not have `slotBefore` or `label`

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Comment by dongsuwon** [comment]
LGTM

**Pull request merged**